core/subscriptions.go

```diff
@@ -26,8 +26,6 @@ func subscribe(channel string, client *client) {
 func unsubscribe(channel string, client *client) {
 	filtered := clients{}
 
-	// Probably can't handle unsubscribing from a non-existent channel, but
```

Comment:
IT's not checking anything, it's a comment stating something that hasn't changed since the comment was written.